easy-ui-react/src/Button/Button.stories.mdx

```diff
@@ -0,0 +1,157 @@
+import { Canvas, Meta, Story, ArgsTable } from "@storybook/addon-docs";
+import { createLabelledOptionsControl } from "../utilities/storybook";
+import { Button } from "./Button";
+import { Icon } from "../Icon";
+import AddIcon from "@easypost/easy-ui-icons/Add";
+import CheckCircleIcon from "@easypost/easy-ui-icons/CheckCircle";
+import InfoIcon from "@easypost/easy-ui-icons/Info";
+import ArrowBackIcon from "@easypost/easy-ui-icons/ArrowBack";
+
+<Meta
+  title="Button"
+  argTypes={{
+    children: {
+      control: "text",
+    },
+  }}
+  component={Button}
+/>
+
+export const Template = (args) => <Button {...args} />;
+
+export const buttonContainerStyles = {
+  display: "flex",
+  alignItems: "center",
+  gap: "12px",
+  flexWrap: "wrap",
+};
+
+export const inverseBackground = {
+  backgroundColor: "var(--ezui-color-blue-800)",
+  padding: "12px",
+};
+
+# Button
```

Comment:
cleaned up button stories, i wasn't able to nest at the story level, but open to suggestions about a better grouping

Reply:
i find this update much better